src/libraries/System.Linq.AsyncEnumerable/src/System.Linq.AsyncEnumerable.csproj

```diff
@@ -3,7 +3,13 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <IsPackable>true</IsPackable>
-    <NoWarn>$(NoWarn);CS1998</NoWarn> <!-- Various IAsyncEnumerable implementations that don't need to use await -->
+
+    <!-- Various IAsyncEnumerable implementations that don't need to use await -->
+    <NoWarn>$(NoWarn);CS1998</NoWarn>
```

Comment:
Is there a possibility that this could result in legitimate warnings slipping through in the future? I would assume that the answer is no given the simple nature of this library.

Reply:
> I would assume that the answer is no given the simple nature of this library.

Right, my thinking as well